src/MSBuild/XMake.cs

```diff
@@ -4269,7 +4269,7 @@ private static LoggerDescription ParseLoggingParameter(string parameter, string
             }
             catch (Exception e) when (loggerDescription.IsOptional)
             {
-                Console.WriteLine(ResourceUtilities.FormatResourceStringStripCodeAndKeyword("OptionalLoggerCreationMessage", e.Message));
+                Console.WriteLine(loggerDescription.Name ": " + ResourceUtilities.FormatResourceStringStripCodeAndKeyword("OptionalLoggerCreationMessage", e.Message));
```

Comment:
Could you change the string itself, rather than prepending this?

Reply:
Exception thrown is generic "The given assembly name was invalid." and resource string is generic localized string ("something went wrong"). So there is no venue to include details of loggerDescription .
I can add separate variable formatting the line before invoking Console.WriteLine(). I'm not familiar with codebase enough, not sure if there is preferred style.

Reply:
I meant change the resource string--pushed a commit that does that to look at.